### PR TITLE
Update libraries part 2 

### PR DESCRIPTION
**Pull request by lciolecki**
-  Update react-router to 5.x

### CODE REVIEW

File: ui/client/containers/Archive.js [hunk 1]
```diff
@@ -41,14 +40,6 @@ class Archive extends React.Component {
     }
   }
 
-  unarchiveProcess(processId) {
```
Comment (arkadius) [inline ui/client/containers/Archive.js]: what happened with that?

File: ui/client/components/modals/CalculateCountsDialog.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ import {dateFormat} from "../../config";
 class CalculateCountsDialog extends React.Component {
   //React Datepicker supports different time format
   dateFormat="yyyy-MM-dd HH:mm:ss"
+  timeFormat="HH:mm:ss"
```
[the remark this answers is not in the document]
Comment (lciolecki) [reply ui/client/components/modals/CalculateCountsDialog.js]: Yes, they can put seconds by finger.. :)

File: ui/client/containers/Archive.js (hunk 1 above)
Comment (lciolecki) [reply ui/client/containers/Archive.js]: This was unused.

File: ui/client/common/SVGUtils.js [hunk 3]
```diff
@@ -48,6 +48,17 @@ class SVGUtils {
     }
 
   }
+
+  svgToDataURL = svgStr => {
+    const encoded = encodeURIComponent(svgStr)
```
[the remark this answers is not in the document]
Comment (lciolecki) [reply ui/client/common/SVGUtils.js]: It's okej. There is no big difference between  encodeURI vs encodeURIComponent.

[the remark this answers is not in the document]
Comment (bottaio) [reply ui/client/common/SVGUtils.js]: Or at least add comment explaining why you have to do it this way.

File: ui/client/components/graph/NodeDetailsContent.js [hunk 4]
```diff
@@ -538,14 +538,13 @@ export class NodeDetailsContent extends React.Component {
   nodeValueClass = (isMarked) => "node-value" + (isMarked ? " marked" : "");
 
   setNodeDataAt = (propToMutate, newValue, defaultValue) => {
-    if (_.isEmpty(newValue) && !_.isUndefined(defaultValue)) {
-      newValue = defaultValue;
-    }
+    const value = newValue == null ? defaultValue : newValue
```
[the remark this answers is not in the document]
Comment (lciolecki) [reply ui/client/components/graph/NodeDetailsContent.js]: newValue == null is exacly the same:  newValue == null or newValue == undefinde.

[the remark this answers is not in the document]
Comment (lciolecki) [reply ui/client/components/graph/NodeDetailsContent.js]: Hmm, mayby you are right - I will add this.

File: ui/client/common/SVGUtils.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (lciolecki) [reply ui/client/common/SVGUtils.js]: >
The difference between encodeURI and encodeURIComponent is encodeURIComponent encodes the entire string, where encodeURI ignores protocol prefix ('http://') and domain name. encodeURIComponent is designed to encode everything, where encodeURI ignores a URL's domain related roots.
> 
So I think better is use encodeURIComponent.

**Pull request merged**